hail/python/hailtop/batch/backend.py

```diff
@@ -803,19 +793,23 @@ async def compile_job(job):
         if not wait:
             print(f'Submitted batch {batch_handle.id}, see {url}')
         if verbose:
-            print(f'Batch consists of {n_jobs_submitted} jobs, submitted in {round(time.time() - submit_batch_start, 3)} seconds:')
+            print(f'Submitted batch {batch_id} with {n_jobs_submitted} jobs in {round(time.time() - submit_batch_start, 3)} seconds:')
             for jid, cmd in jobs_to_command.items():
                 print(f'{jid}: {cmd}')
             print('')
+
+        deploy_config = get_deploy_config()
+        url = deploy_config.external_url('batch', f'/batches/{batch_id}')
+
```

Comment:
I think these lines are a merge accident.

Reply:
Bumping this (minor) comment.

Reply:
Thanks @jmarshall, have resolved by removing these lines